src/main/java/com/scalar/db/storage/multistorage/MultiStorageDatabaseConfig.java

```diff
@@ -0,0 +1,117 @@
+package com.scalar.db.storage.multistorage;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableMap.Builder;
+import com.scalar.db.config.DatabaseConfig;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Map;
+import java.util.Properties;
+import javax.annotation.concurrent.Immutable;
+
+@Immutable
+public class MultiStorageDatabaseConfig {
```

Comment:
```suggestion
public class MultiStorageConfig {
```
MultiStorage should accept MultiStorageConfig since JdbcDatabase accepts JdbcDatabaseConfig?